### ISSUE TITLE
lasso or box select

### ISSUE THREAD

**Issue by llaenowyd** [feathericons/feather#943]
## Icon Request

* Icon name: select (lasso, box)
* Use case: I noticed there is a crop icon.  I always want to box select before crop.  I can't be the only one
* Screenshots of similar icons:  

![select_icons](https://user-images.githubusercontent.com/41239783/85816907-883e6c80-b73a-11ea-813e-6779f502bf59.jpeg)
![blender_menu](https://user-images.githubusercontent.com/41239783/85816913-8aa0c680-b73a-11ea-8058-c6781f8d21ae.png)
![ancient_photoshop_select_icons](https://user-images.githubusercontent.com/41239783/85816923-8d032080-b73a-11ea-9762-0736eee6402c.png)
<img width="95" alt="gimp_with_box_and_lasso_selects" src="https://user-images.githubusercontent.com/41239783/85816926-8f657a80-b73a-11ea-89c4-9f173ecf3138.png">

**Comment by llaenowyd** [comment]
🦗  I infer that I am in fact the only one.

**Comment by locness3** [comment]
You aren't but it takes time for an issue to be considered.

**Comment by mishkaio** [comment]
![box and lasso feather preview](https://user-images.githubusercontent.com/60020191/87992838-b5b6d580-caae-11ea-8d10-d28678a7d81a.png)

[feather-box-lasso.zip](https://github.com/feathericons/feather/files/4950313/feather-box-lasso.zip)

I'm not 100% certain that these satisfy the design guidelines requirement about optical volumes but the lasso having the dangling bit really restricts the height of the main loop. I also don't know the right way to add these but I thought replying to the request thread was a good start.

**Comment by locness3** [comment]
You can make a pull request here, but it might not get merged soon. You could submit it at @featherity .

**Comment by llaenowyd** [comment]
bump - @mishkaio could you submit a PR with those icons, or link to it if you have?

**Comment by mishkaio** [comment]
@llaenowyd yes, I can attempt to do that. At this risk of sounding ridiculous, I never have before but I want to be involved so I need to figure it out...

**Comment by mishkaio** [comment]
@llaenowyd I bugged a kind dev coworker to get enough info to do this, hopefully this is what you needed.
[https://github.com/featherity/featherity/pull/57](url)

**Comment by llaenowyd** [comment]
above link broken, should be [https://github.com/featherity/featherity/pull/57](https://github.com/featherity/featherity/pull/57)
(link text is correct but markdown mixed up)
